### PR TITLE
Add handling for metric relabeling rules to cAdvisor.

### PR DESCRIPTION
**Pull request by petewall**
Also remove discovery rules for things that don't support it (because they use the alloy module for discovery).

### CODE REVIEW

**Comment by petewall** [comment]
cc @fplantinga-guida 
Can you send me an email address to tag you on this? It can be the GitHub  no-reply email, too.

File: charts/feature-cluster-metrics/templates/_node_exporter.alloy.tpl [hunk 1]
```diff
@@ -58,7 +58,9 @@ node_exporter.scrape "metrics" {
 prometheus.relabel "node_exporter" {
   max_cache_size = {{ (index .Values "node-exporter").maxCacheSize | default .Values.global.maxCacheSize | int }}
 
+{{- if (index .Values "node-exporter").extraMetricProcessingRules }}
   {{(index .Values "node-exporter").extraMetricProcessingRules}}
```
Comment (skl) [inline charts/feature-cluster-metrics/templates/_node_exporter.alloy.tpl:62]: Only this one is missing `| indent 2`, not sure if that matters

**Review by skl — APPROVED** [review]

File: charts/feature-cluster-metrics/templates/_node_exporter.alloy.tpl (hunk 1 above)
Comment (petewall) [reply charts/feature-cluster-metrics/templates/_node_exporter.alloy.tpl:62]: Whoa, expert catch! Thanks

**Comment by fplantinga-guida** [comment]
> cc @fplantinga-guida Can you send me an email address to tag you on this? It can be the GitHub no-reply email, too.

76436660+fplantinga-guida@users.noreply.github.com

**Pull request merged**